81E00600.4D0200.dts

```diff
@@ -0,0 +1,132 @@
+
+/dts-v1/;
+
+/ {
+        name = "";
+        compatible = "ibm,opal-in-memory-counters";
+        #address-cells = <0x1>;
+        #size-cells = <0x1>;
+        ima-nest-offset = <0x320000>;
+        ima-nest-size = <0x30000>;
+        version-id = "";
+
+        mcs {
```

Comment:
Change the name of the "mcs" node to something like "mcs-0". In future, if we add any more mcs units, then we can accommodate similar nodes with this design.